src/app/browse-page/browse-groups/groups.component.ts

```diff
@@ -337,8 +341,62 @@ export class BrowseGroupsComponent implements OnInit {
     .take(1)
     .subscribe(
       (data)  => {
+        // Set the group level to show in the number of result message
+        let groupLabel : string = ''
+        switch (browseLevel) {
+          case 'all': {
+            groupLabel = 'All groups'
+            break
+          }
+          case 'created': {
+            groupLabel = 'My groups'
+            break
+          }
+          case 'private': {
+            groupLabel = 'Private groups'
+            break
+          }
+          case 'shared_by_me': {
+            groupLabel = 'Shared by Me groups'
+            break
+          }
+          case 'institution': {
+            groupLabel = 'Institutional groups'
+            break
+          }
+          case 'shared': {
+            groupLabel = 'Shared with Me groups'
+            break
+          }
+          case 'public': {
+            groupLabel = 'Artstor Curated groups'
+            break
+          }
+          default: {
+              break
+          }
+        }
+
+        // Set the number of result message
+        if (data.total !== 0){
+          if (this.pagination.size < data.total) {
+            this.numResultMsg = this.pagination.size + ' of ' + data.total + ' results for \"' + this.searchTerm + '\"' + ' from <i>' + groupLabel + '</i>.'
```

Comment:
As per the new design changes from Kelsey - we should be displaying just the number of results instead of number of results out of total number of results for keyword from filter group i.e. `10 results for "Lecture 1" from Institutional groups`.

Reply:
Just asked Kelsey about this, she conforms that it's better to show `10 of 10 results`. The design is different from the content of the ticket and we agree to follow the ticket.

Reply:
So, I just realized that Jialin's implementation is correct, my design is not. I'm still learning the ropes! Thanks, the design looks good.

Reply:
Okay cool. I think its good to go then. Gonna merge the changes!